docs/source/deployment/CORS_Headers.rst

```diff
@@ -0,0 +1,95 @@
+========================================
+Add CORS Headers to MOTECH Configuration
+========================================
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Add CORS support to Apache server
+=================================
+
+First step is enable mod headers for Apache. Do it by running the following command:
+
+    a2enmod headers
+
+Configuration CORS, if you want set this globally, add following line into apache server configuration (usually /etc/apache*.conf)
+into <directory **PATH**> </directory>
+
+.. code-block:
+
+	Header always set Access-Control-Allow-Origin "*"
+	Header always set Access-Control-Allow-Headers **REQUEST HEADERS**
+	Header always set Access-Control-Expose-Headers **RESPONSE HEADERS**
+	Header always set Access-Control-Allow-Methods **METHODS**
+
+**REQUEST HEADERS** is the list of headers separated by comma that can be used to prepare request. Example:
+ "Content-Type,X-Requested-With,accept,authorization,Origin,Access-Control-Request-Method,Access-Control-Request-Headers"
```

Comment:
so, what do I enter here? It's not clear from the description - do I choose some or do I have to use them all to have Motech functioning properly. If the latter is true, then why is this called an example?